src/orquestra/sdk/_ray/_dag.py

```diff
@@ -533,5 +534,35 @@ def list_workflow_runs(
             ]
         return wf_runs
 
+    def list_workflow_run_summaries(
+        self,
+        *,
+        limit: t.Optional[int] = None,
+        max_age: t.Optional[timedelta] = None,
+        state: t.Optional[t.Union[State, t.List[State]]] = None,
+        workspace: t.Optional[WorkspaceId] = None,
+    ):
+        """
+        List summaries of the workflow runs, with some filters
+
+        Args:
+            limit: Restrict the number of runs to return, prioritising the most recent.
+            max_age: Only return runs younger than the specified maximum age.
+            status: Only return runs of runs with the specified status.
+            workspace: Only return runs from the specified workspace.
+
+        Returns:
+            A list of the workflow runs' summaries.
```

Comment:
**suggestion**: please use a return type hint in the signature instead.

Reply:
[d62755f](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/313/commits/d62755f55a82b6a65209223d699ad6bceff7f906)